test/integration/imagechange_buildtrigger_test.go

```diff
@@ -465,6 +465,8 @@ func TestMultipleImageChangeBuildTriggers(t *testing.T) {
 	}
 	defer watch2.Stop()
 
+	ignoreBuilds := make(map[string]string)
+
 	for _, tc := range triggersToTest {
 		imageStream := mockImageStream(tc.name, tc.tag)
 		imageStreamMapping := mockStreamMapping(tc.name, tc.tag)
```

Comment:
Trying to understand this test... are these really mocks, or just poor naming?
It seems to me that those are factory functions and we create real image streams in the server, am I wrong?